### PR TITLE
Update static text

### PR DESCRIPTION
**Pull request by simaob**
This PR:

- updates partners description;
- updates order of logos on the partners page;
- adds link to partnership page from the partners page;
- adds more links to the settings page for FAQ and ContactUs page, and updates method to link to these.
- adds contact-us components based on the Partners components.

⚠️ Warning⚠️ : I don't have the project properly setup on my machine, so I couldn't run this to see if this is working as it should. Please make sure to test it. if this works well then I can add the text for FAQ and Terms and Conditions that they have also provided.

### CODE REVIEW

**Comment by j8seangel** [comment]
I've fixed some broken links and included the contact us screen to be able to navigate there, thank you very much @simaob !

**Comment by hectoruch** [comment]
This PR include new Terms and Conditions list and detail view.

terms

**Comment by hectoruch** [comment]
This PR include new FAQ list and detail view.

faq

**Comment by hectoruch** [comment]
Include links on texts using [react-native-hyperlink](https://github.com/obipawan/react-native-hyperlink)

links

**Comment by hectoruch** [comment]
Change and update texts.
- Task -> https://www.pivotaltracker.com/story/show/150638948

File: app/components/settings/contact-us/index.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+        showsHorizontalScrollIndicator={false}
+      >
+        <View style={styles.contactUs}>
+          <Hyperlink
+            linkDefault linkStyle={{ color: '#97be32' }}
+            linkText={(url) => (url === 'mailto:forestwatcher@wri.org' ? 'forestwatcher@wri.org' : url)}
+          >
+            <Text
+              style={styles.contactUsText}
+              onPress={() => Linking.openURL('mailto:forestwatcher@wri.or?subject=FW app contact')}
```
Comment (sorodrigo) [inline app/components/settings/contact-us/index.js]: Do we need the onPress handler if we are using Hyperlink?

File: app/components/settings/faq/detail/index.js [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+
+  state = {
+    description: this.props.contentFaq[0].description,
+    orderList: this.props.contentFaq[1].orderList,
+    orderListLetters: this.props.contentFaq[2].orderListLetters,
+    footerText: this.props.contentFaq[3].footerText
+  };
+
+  componentDidMount() {
+    // tracker.trackScreenView('TermsAndConditions');
```
Comment (sorodrigo) [inline app/components/settings/faq/detail/index.js]: Maybe we should also track this viewpage with something like FAQ

File: app/components/settings/faq/detail/index.js [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+        style={styles.container}
+        contentContainerStyle={styles.containerContent}
+        showsVerticalScrollIndicator={false}
+        showsHorizontalScrollIndicator={false}
+      >
+
+        <View
+          style={styles.faq}
+        >
+          {this.state.description !== null ? this.state.description.map((data, key) => (
```
Comment (sorodrigo) [inline app/components/settings/faq/detail/index.js]: Let's talk about this

File: app/components/settings/faq/index.js [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+    topBarElevationShadowEnabled: false,
+    navBarBackgroundColor: Theme.background.main
+  };
+
+  static propTypes = {
+    navigator: PropTypes.object.isRequired
+  };
+
+  componentDidMount() {
+    // tracker.trackScreenView('TermsAndConditions');
```
Comment (sorodrigo) [inline app/components/settings/faq/index.js]: I guess that we could remove the tracker from FAQ-detail and only track this one with FAQ

File: app/components/settings/faq/detail/index.js [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+              <Hyperlink linkDefault linkStyle={{ color: '#97be32' }}>
+                <Text style={styles.faqText}>{data.text}</Text>
+              </Hyperlink>
+            </View>
+          )) : <Text>{}</Text>}
+
+          {this.state.orderListLetters !== null ? this.state.orderListLetters.map((data, key) => (
+            <View key={key}>
+              <View style={styles.faqListLetter}>
+                <Text style={styles.faqText}>{String.fromCharCode(97 + (key % 27))}. </Text>
```
Comment (sorodrigo) [inline app/components/settings/faq/detail/index.js]: this is pretty cool

File: app/components/settings/faq/detail/index.js [hunk 6]
```diff
@@ -0,0 +1,102 @@
...
+    topBarElevationShadowEnabled: false,
+    navBarBackgroundColor: Theme.background.main
+  };
+
+  static propTypes = {
+    contentFaq: PropTypes.any
+  };
+
+  state = {
+    description: this.props.contentFaq[0].description,
```
Comment (sorodrigo) [inline app/components/settings/faq/detail/index.js]: we should use an object instead of an array

File: app/components/settings/faq/index.js [hunk 7]
```diff
@@ -0,0 +1,142 @@
...
+  }
+
+  handleStaticLinks = (section, text, list) => {
+    this.props.navigator.push({
+      screen: section,
+      title: text,
+      passProps: {
+        contentFaq: list === '' ? null : list
```
Comment (sorodrigo) [inline app/components/settings/faq/index.js]: why empty string? seems odd

File: app/components/settings/terms-and-conditions/detail/index.js [hunk 8]
```diff
@@ -0,0 +1,71 @@
...
+    descriptionTerm: PropTypes.any
+  };
+
+  state = {
+    content: this.props.contentTerm,
+    description: this.props.descriptionTerm
+  };
+
+  componentDidMount() {
+    // tracker.trackScreenView('TermsAndConditions');
```
Comment (sorodrigo) [inline app/components/settings/terms-and-conditions/detail/index.js]: Lets remove this

File: app/components/settings/terms-and-conditions/index.js [hunk 9]
```diff
@@ -0,0 +1,111 @@
...
+    topBarElevationShadowEnabled: false,
+    navBarBackgroundColor: Theme.background.main
+  };
+
+  static propTypes = {
+    navigator: PropTypes.object.isRequired
+  };
+
+  componentDidMount() {
+    // tracker.trackScreenView('TermsAndConditions');
```
Comment (sorodrigo) [inline app/components/settings/terms-and-conditions/index.js]: Lets maintain this one

File: package.json [hunk 10]
```diff
@@ -25,6 +25,7 @@
     "geopoint": "1.0.1",
     "jsonapi-serializer": "3.5.3",
     "kdbush": "1.0.1",
+    "linkify-it": "^2.0.3",
```
Comment (sorodrigo) [inline package.json]: lets remove this one if its not needed anymore

File: package.json [hunk 11]
```diff
@@ -73,7 +75,7 @@
     "eslint-plugin-import": "2.2.0",
     "eslint-plugin-jsx-a11y": "2.2.3",
     "eslint-plugin-react": "6.8.0",
-    "husky": "0.13.3",
+    "husky": "^0.13.3",
```
Comment (sorodrigo) [inline package.json]: careful with this changes

File: app/components/settings/contact-us/index.js (hunk 1 above)
Comment (hectoruch) [reply app/components/settings/contact-us/index.js]: 👍

**Pull request merged**